pages/contacts/[id].js

```diff
@@ -14,6 +14,7 @@ const StyledTitleText = styled.h1`
 `;
 
 export default function Details() {
+  const [data, setData] = useLocalStorageState("friendsApp");
```

Comment:
Same here as mentioned above for data
```suggestion
export default function Details({data}) {

```